rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -107,6 +91,8 @@ def _precompiled_apple_resource_bundle_impl(ctx):
         paths.join("%s-intermediates" % ctx.label.name, "Info.plist"),
     )
 
+    # No need to pass swift_module to merge_root_infoplists
+    partials_args.pop("swift_module")
```

Comment:
What happens if we do pass this?

Reply:
@luispadron `merge_root_infoplists` fails because swift_module is unexpected param. I could make new dictionary but not sure if it's better solution